src/ast/mod.rs

```diff
@@ -411,6 +411,7 @@ pub enum Statement {
     },
     /// `CREATE SOURCES`
     CreateSources {
+        like: Option<LikeFilter>,
```

Comment:
I'm thinking maybe this should be an `Option<String>` instead of an `Option<LikeFilter>`—unless you were thinking about adding additional filter options, beside `LIKE`, in the future?